examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<p>May you also add an example of how to use <code>to_hierarchical_dataset</code> to create df_exog </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>

Reply:
<p>Do we really need this one ? We lready have an exampe for this method. It will be just the same manipulations</p>

Reply:
<p>We have and additional important flags for df_exog case</p>